packages/field-base/src/validate-mixin.d.ts

```diff
@@ -16,6 +16,17 @@ export declare class ValidateMixinClass {
    */
   invalid: boolean;
 
+  /**
+   * Set to true to enable manual validation mode. This mode disables automatic
+   * constraint validation, allowing you to control the validation process yourself.
+   * You can still trigger constraint validation manually with the `validate()` method
+   * or use `checkValidity()` to assess the component's validity without affecting
+   * the invalid state. In manual validation mode, you can also manipulate
+   * the `invalid` property directly through your application logic without conflicts
+   * with the component's internal validation.
+   */
+  manualValidation: boolean;
```

Comment:
Let's add `@attr {boolean} manual-validation` annotation also to this comment.

Reply:
Done.